AttitudeManager/FlightModes/Src/fbwa.cpp

```diff
@@ -0,0 +1,23 @@
+#include "fbwa.hpp"
+#include "CommonDataTypes.hpp"
+#include "AM.hpp"
+
+namespace AM {
+    AttitudeManagerInput FBWA::run(const AttitudeManagerInput& input) {
+        
+        AttitudeManagerInput mappedOutputs;
+
+        mappedOutputs.pitch = AM::AttitudeManager::PITCH_LIM_MIN + (input.pitch - AM::AttitudeManager::PITCH_LIM_MIN) * (AM::AttitudeManager::PITCH_LIM_MAX - AM::AttitudeManager::PITCH_LIM_MIN) / (AM::AttitudeManager::INPUT_MAX - AM::AttitudeManager::INPUT_MIN);
```

Comment:
My bad on this one - we should be grabbing the limits from the flight mode definition.

Take a look at: https://github.com/UWARG/efs-zeropilot-3.5/blob/57923d154edc9d4b52c8b9a831db0f8e842a4d5c/Config_Models/example_model/config.hpp#L102, we want to import these control limits in our model constructor somehow, and use them when doing this.

Also - if you could please update your PR to include comments explaining the math that you used :)